docker/nightly/centos7/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM centos:latest
+
+LABEL maintainer="Jumping Yang <yzpcreate@qq.com>"
+LABEL readme.md="https://github.com/Microsoft/omi/blob/master/docker/README.md"
+LABEL description="This Dockerfile will install the latest code of OMI."
+
+ARG fork=Microsoft
+ARG branch=master
+
+SHELL ["/bin/bash", "-c"] 
+RUN yum update -y; \
+    yum install git bind-utils gcc-c++ rpm-devel pam-devel openssl-devel rpm-build krb5-devel redhat-lsb-core;
+RUN git clone https://github.com/$env:fork/omi.git -b $env:branch; \
+    cd omi; \
+    pushd Unix; \
+    ./configure --enable-system-build --enable-native-kits; \
```

Comment:
I don't think this would be quite right for a "local" version of OMI. In fact, this would be totally unsupported. If you want to grab the latest supported version of OMI, you should be grabbing it from [omi-kits](https://github.com/Microsoft/omi-kits) repo.

The version you're building here is untested, unsupported (except on specific build platforms), and certainly not suggested.